tmt/cli.py

```diff
@@ -1207,19 +1207,19 @@ def headfoot(text: str) -> None:
 @click.pass_context
 @filter_options_long
 @story_flags_filter_options
-@click.option(
+@option(
     '-h', '--how', metavar='METHOD', default=_story_export_default, show_default=True,
     help='Output format.',
     type=click.Choice(choices=_story_export_formats))
-@click.option(
+@option(
     '--format', metavar='FORMAT', default=_story_export_default, show_default=True,
     help='Output format. Deprecated, use --how instead.',
```

Comment:
This can use the new `deprecated` option? :)

Reply:
Yes, but since there are several deprecated options around the tmt code base, I wanted to switch them to `deprecated` in a separate patch, to avoid mixing two conversion steps into one PR.